admin_interface/templates/admin/filter.html

```diff
@@ -8,15 +8,14 @@
 
 {% else %}
 
-    {# Use the default list filter template -> https://github.com/django/django/blob/master/django/contrib/admin/templates/admin/filter.html #}
+    {# Use a changed default list filter template -> https://github.com/django/django/blob/master/django/contrib/admin/templates/admin/filter.html #}
 
-    <h3>{% blocktrans with filter_title=title %} By {{ filter_title }} {% endblocktrans %}</h3>
-    <ul>
+    <h3 {% if spec.lookup_val or spec.value %}class="active"{% endif %}>{% blocktrans with filter_title=title %} By {{ filter_title }} {% endblocktrans %}</h3>
+    <ul {% if spec.lookup_val or spec.value %}class="active"{% endif %}>
```

Comment:
Some custom filters don’t have `lookup_val`, so this calls the normal `value` method.
`value` alone is not enough so I kept both; performance impact should be minimal.